src/test/operators/table_scan_test.cpp

```diff
@@ -52,7 +52,9 @@ class OperatorsTableScanTest : public BaseTest, public ::testing::WithParamInter
     _int_int_partly_compressed->execute();
   }
 
-  std::shared_ptr<TableWrapper> load_and_encode_table(const std::string& path, const ChunkOffset chunk_size = 2) {
+  std::shared_ptr<TableWrapper> load_and_encode_table(
+      const std::string& path, const ChunkOffset chunk_size = 2,
+      const std::optional<std::pair<ColumnID, OrderByMode>> ordered_by = std::nullopt) {
```

Comment:
...and so starts the invasion of flags, see? Totally not fatal here but just to underlie the point made in https://github.com/hyrise/hyrise/issues/1519#issuecomment-469148314: Why would something called `load_and_encode_table` spend a significant portion of its lines doing something (marking chunks as sorted) that is not its job?

Reply:
What are you implying here? Should we just rename `load_and_encode_table` or should we do this at a completely different place?

Reply:
Not implying the necessity of a change here, just pointing out that flags as discussed in the linked issue tend to infest places that shouldn't be concerned with them

Reply:
Ok. Thanks for the clarification